salt/config/__init__.py

```diff
@@ -1685,6 +1684,10 @@ def _read_conf_file(path):
     '''
     Read in a config file from a given path and process it into a dictionary
     '''
+    if (not salt.utils.validate.path.exists(path, quiet=os.path.basename(path) == '.saltrc')
```

Comment:
Because it is optional and forgotten entity (and normally never used), this is why I muted it to the logs.